platform/darwin/src/NSBundle+MGLAdditions.m

```diff
@@ -35,10 +35,15 @@ + (nullable NSString *)mgl_frameworkBundleIdentifier {
 
 + (nullable NSString *)mgl_applicationBundleIdentifier {
     NSString *bundleIdentifier = [NSBundle mainBundle].bundleIdentifier;
+    
+#if (defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && (__IPHONE_OS_VERSION_MAX_ALLOWED < 120200)) || \
+    (defined(__MAC_OS_X_VERSION_MAX_ALLOWED) && (__MAC_OS_X_VERSION_MAX_ALLOWED < 101404))
+    // Before SDK 12.2 (bundled with Xcode 10.2): There’s no main bundle identifier when running in a unit test bundle.
+    // 12.2 and after: the above bundle identifier is: com.apple.dt.xctest.tool
```

Comment:
If there is a bundle identifier in the XCTest case, then the conditional below wouldn’t run anyways. Did the conditional also cover other cases where a main bundle might be missing? There should be no harm in leaving it in.

Reply:
I chose to add the #if for clarity, and also to trigger a potential crash in the future if there are changes.